tailwind.config.js

```diff
@@ -143,6 +143,13 @@ module.exports = {
       },
     },
     extend: {
+      height: {
+        '128': '32rem',
+        '192': '48rem',
+        '212': '53rem',
```

Comment:
in design we use 8 point grid system, that means that all numbers should be devidable by 8. we should never use odd numbers.

Reply:
I've added some more custom sizes with 8 point grid system and fixed card h